src/components/Table/__tests__/__snapshots__/Table.test.tsx.snap

```diff
@@ -167,6 +169,7 @@ exports[`Table Shows Table with default props 1`] = `
           Numbers
           <svg
             class="c4"
+            direction="1"
```

Comment:
the browser will complain that `direction` isn't a valid attribute of `svg`, I think there's a way to add `data-` prefix or something similar to make it happy. This needs to be done in other components but might as well test it here